src/platform/datapath_raw.c

```diff
@@ -419,15 +419,122 @@ CxPlatSendDataIsFull(
     return TRUE;
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+inline
+QUIC_STATUS
+CxPlatResolveRoute(
```

Comment:
Are `GetBestRoute2`, `GetIfEntry2`, and `GetIpNetEntry2` all individual syscalls to kernel? Which, if any, could theoretically block on network IO if the info isn't cached?

Reply:
I'm pretty sure the only call here that blocks on network IO is ResolveIpNetEntry2.

Reply:
And yes, each of these is a syscall.

Reply:
Let's please add comments indicating which functions may block.